ls1intum/Artemis
# `Exam mode`: Fix missing translation when deleting exams

### Checklist
#### General
#### Client

### Motivation and Context
Closes #4854

### Description
I added the missing translation

### Steps for Testing
1. Delete an exam
2. Check that a correct translation is shown

### Review Progress
#### Code Review
#### Manual Tests

### Screenshots
![grafik](https://user-images.githubusercontent.com/26540346/159947275-b4ab7b06-5da6-4f3b-aab7-8fd2f1aedc11.png)
